SimDataFormats/Associations/src/classes_def.xml

```diff
@@ -141,4 +141,39 @@
 
   <class name="edm::Ref<std::vector<TTTrackTruthPair<edm::Ref<edm::DetSetVector<Phase2TrackerDigi>,Phase2TrackerDigi,edm::refhelper::FindForDetSetVector<Phase2TrackerDigi> > > > >" />
   <class name="edm::Wrapper<std::vector<TTTrackTruthPair<edm::Ref<edm::DetSetVector<Phase2TrackerDigi>,Phase2TrackerDigi,edm::refhelper::FindForDetSetVector<Phase2TrackerDigi> > > > >" />
+
+
+  
+  <class name="std::pair<FTLClusterRef, std::vector<MtdSimLayerClusterRef> >" persistent="true" />
```

Comment:
While `persistent="true"` would technically be ok, I'd suggest to remove it and rely it being the default. This would make it easier to focus the reader's eye on the `persistent="false"` cases.